api/tests/Feature/PoolTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+use App\Models\Classification;
+use App\Models\CmoAsset;
+use App\Models\Pool;
+use App\Models\PoolCandidate;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Nuwave\Lighthouse\Testing\ClearsSchemaCache;
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Tests\TestCase;
+use Database\Helpers\ApiEnums;
+
+class PoolTest extends TestCase
+{
+  use \Illuminate\Foundation\Testing\RefreshDatabase;
+  use MakesGraphQLRequests;
+  use ClearsSchemaCache;
+
+  protected function setUp(): void
+  {
+    parent::setUp();
+    $this->bootClearsSchemaCache();
+  }
+
+  public function testPoolAdvertisementAccessor(): void
+  {
+    // Create new pools and attach to new pool candidates.
+    $pool1 = Pool::factory()->create([
+        'id' => 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11',
```

Comment:
By the time you read this, Matt's PR with a nice UUID generator will probably be merged that you can use.
https://github.com/GCTC-NTGC/gc-digital-talent/pull/3211/files#diff-725a56c335acceced5cc990078f4a89c89b1a54e323037baf43b7758a3bbc0d4

Reply:
ehh, I kinda like it as is since you need to use the id immediately